backend/api/views/milestone.py

```diff
@@ -25,14 +25,15 @@ def get_user_milestones(self, request):
         username = self.request.user.username
         list_of_milestones = []
         if username:
-            projects = Project.objects.filter(owner__username=username)
-            serializer = ProjectGETPublicSerializer(projects, many=True)
+            projects = Project.objects.filter(members__username__in=[username])
+            serializer = ProjectMilestonesSerializer(projects, many=True)
             for item in serializer.data:
                 for milestone in item['milestones']:
+                    milestone['project_name'] = item['name']
```

Comment:
What is this line for exactly?

Reply:
name of the project that milestone belongs to

```
"result": [
    {
      "id": 1,
      "description": "Milestone ...",
      "date": "2020-12-24",
      "project": 4,
      "project_name": "Project ..."
    },
    {
      "id": 2,
      "description": "Milestone ....",
      "date": "2020-12-31",
      "project": 4,
      "project_name": "Project ..."
    },
    {
      "id": 3,
      "description": "Milestone ...",
      "date": "2020-12-31",
      "project": 5,
      "project_name": "Project ..."
    }
  ]
```